backend/census_historical_migration/sac_general_lib/report_id_generator.py

```diff
@@ -1,15 +1,20 @@
+from ..transforms.xform_string_to_string import string_to_string
 from ..sac_general_lib.utils import (
     xform_census_date_to_datetime,
 )
 
 
 def xform_dbkey_to_report_id(audit_header):
+    """Constructs the report ID from the DBKEY.
+    The report ID is constructed as follows:
+    YYYY-MM-CENSUS-DBKEY."""
     # month = audit_header.fyenddate.split('-')[1]

```

Comment:
We should remove/clean up any comments that are lingering at this point, or make sure they're accurate.

Reply:
Captured in https://github.com/GSA-TTS/FAC/issues/2837